storm-server/src/main/java/org/apache/storm/localizer/AsyncLocalizer.java

```diff
@@ -110,6 +111,7 @@
         this.blobCacheUpdateDuration = metricsRegistry.registerTimer("supervisor:blob-cache-update-duration");
         this.blobLocalizationDuration = metricsRegistry.registerTimer("supervisor:blob-localization-duration");
         this.numBlobUpdateVersionChanged = metricsRegistry.registerMeter("supervisor:num-blob-update-version-changed");
+        this.localResourceFileNotFound = metricsRegistry.registerMeter("supervisdor:local-resource-file-not-found");
```

Comment:
spelling of supervisor.

Also please update ClusterMetrics.md with this metric and any other missing that are in this file.

Reply:
Thanks for catching, updated